README.md

```diff
@@ -31,11 +31,13 @@
 ## SR-IOV Network Device Plugin
 
 The SR-IOV Network Device Plugin is Kubernetes device plugin for discovering and advertising networking resources in the
-form of SR-IOV virtual functions (VFs) and PCI physical functions (PFs) available on a Kubernetes host.
+form of SR-IOV virtual functions (VFs), Auxiliary network devices (in particular Scalable functions) and PCI physical
```

Comment:
```
.... in the form of:
- SR-IOV virtual functions (VFs)
- PCI physical functions (PFs)
- Auxiliary network functions, in particular scalable functions (SF)

which are available on a kubernetes host
```

@DmytroLinkin  should we refer in this document to scalable functions or sub functions ?
one is vendor implementation and the other is the upstream term.

we should use upstream term throughout if possible.

Reply:
once we define here the abberviation SF, we can use it throughout this document instead of Scalable functions WDYT ?

Reply:
> ```
> ....
> ```
> 
> @DmytroLinkin should we refer in this document to scalable functions or sub functions ? one is vendor implementation and the other is the upstream term.
> 
> we should use upstream term throughout if possible.

Until know I was sure that "scalable" is upstream one. I will change it to "sub" everywhere.